Hunted Mobile/Hunted Mobile/Hunted Mobile/ViewModel/MapViewModel.cs

```diff
@@ -861,5 +895,21 @@ public class MapViewModel : BaseViewModel {
             thiefToBeArrested = mapModel.FindThief(new Location(position));
             IsArrestingThief = thiefToBeArrested != null;
         }
+
+        private async Task<bool> NotifyLeavingBoundary() {
+            return await notificationRepository.Create(
+                mapModel.PlayingUser.UserName + " heeft de spelgrenzen verlaten!",
+                gameModel.Id,
+                mapModel.PlayingUser.Id
+            );
+        }
+
+        private async Task<bool> NotifyEnteringBoundary() {
+            return await notificationRepository.Create(
+                mapModel.PlayingUser.UserName + " bevindt zich weer binnen de spelgrenzen.",
+                gameModel.Id,
+                mapModel.PlayingUser.Id
+            );
+        }
```

Comment:
Beide methodes doen precies hetzelfde, alleen het tekstbericht is anders. Als de tekstwaarde kan worden doorgegeven als een parameter, dan kun je de functie ook hergebruiken voor andere dingen.